back/app/services/timeline_service.rb

```diff
@@ -90,4 +90,9 @@ def timeline_active_on_collection(projects)
       [project.id, active]
     end
   end
+
+  def phase_number(phase)
+    phase_ids = phase.project.phase_ids
```

Comment:
I assumed this because in `project.rb:79` we have `has_many :phases, -> { order(:start_at) }, dependent: :destroy` which is how `phase_ids` gets defined. You think the order is not enforced on the `_ids` version?

Reply:
In the specs, the phases are created in such a way that I think the test would be very likely to fail if the order was not ensured.